tools/psyq-obj-parser/psyq-obj-parser.cc

```diff
@@ -1159,7 +1161,7 @@ Usage: {} input.obj [input2.obj...] [-h] [-v] [-d] [-n] [-p prefix] [-o output.o
     int ret = 0;
 
     for (auto& input : inputs) {
-        PCSX::File* file = new PCSX::File(input);
+        std::unique_ptr<PCSX::File> file(new PCSX::PosixFile(input));
```

Comment:
`PCSX::PosixFile(input);` should do the trick here, can't see any reason to heap allocate this one